client/components/web-preview/content.jsx

```diff
@@ -149,14 +148,14 @@ export class WebPreviewContent extends Component {
 		// remove all textual selections when user gives focus to preview iframe
 		// they might be confusing
 		if ( typeof window !== 'undefined' ) {
-			if ( isFunction( window.getSelection ) ) {
+			if ( typeof window.getSelection === 'function' ) {
 				const selection = window.getSelection();
-				if ( isFunction( selection.empty ) ) {
+				if ( typeof selection.empty === 'function' ) {
 					selection.empty();
-				} else if ( isFunction( selection.removeAllRanges ) ) {
+				} else if ( typeof selection.removeAllRanges === 'function' ) {
 					selection.removeAllRanges();
 				}
-			} else if ( document.selection && isFunction( document.selection.empty ) ) {
+			} else if ( typeof document.selection && document.selection.empty === 'function' ) {
```

Comment:
should this be `document.selection && typeof document.selection.empty === 'function'`?

Reply:
Totally! That's what happens when we do those automatically 🙈  Nice find 😉  Fixed in e7cdc5518805dc8376955ddc50a5b03940fcb0b5